### ISSUE TITLE
No GND Connection?

### ISSUE THREAD

**Issue by ghost** [aauer1/sam9g25#1]
![unbenannt](https://cloud.githubusercontent.com/assets/27053295/24827143/789a3a40-1c45-11e7-8992-6af42b6e95c5.PNG)
I use Eagle 8.1 Premium, and I also activate Route2, but there are no GND Connections on the board. Is it a bug or is it my fault?

**Comment by kinweilee** [comment]
The ground connection is connected. Use the 'SHOW' command (icon looks like an eyeball) and select the 'Layer 2 (GND)' perimeter. The GND on all subsequent items will also highlight.

You can also type-in the commands 'set polygon on' followed by 'ratsnest' to show the ground planes.

**Comment by ghost** [comment]
Thank you for your reply. I use the "SHOW" command and select the "Layer 2 (GND)" perimeter. Some vias get highlighted, but there are still no connections on the board.

When I use your second advice, some GND connections appear and some not.

If I replace the GND connection from X5 in the Schematics, It looks like I get all GND connections on the board, just unrouted.

**Comment by kinweilee** [comment]
I downloaded the original files again, and when I open the document I get the following warning:
```
Loading /Users/lee/Documents/eagle/sam9g25-master/sam9g25.sch ...

Warning(s):

line 14280: ignoring redefinition of pinref to part 'X5', gate 'G$1', pin 'GND'
```

I never paid attention to it before but I believe this relates to your original question. It's probably safe to delete the connection and reattach the GND symbol in the schematic.

Platform: Eagle 7.7.0 OSX 10.11.6

**Comment by ghost** [comment]
Maybe a possible solution would be:
1. Remove line 14280 in the schematic file (with a notepad).
2. Remove and then add the GND connector from X5 in the schematic, so EAGLE shows you unrouted connections on the board.
3. Reconnect them.

But I'm worried about the already exist vias. Because some of them look like there a for GND connections and I think the GND connections exist, but there are invisible. For some reason. I would try to reconnect the GND connections, but I have no experience with BGA. And it's not sure if the board would work then like I sad, I think the GND connections are already there.

By the way, EAGLE shows me the same warning.
Platform: EAGLE 8.1.0 Premium, Windows 8.1 64-Bit

**Comment by kinweilee** [comment]
Deleting the GND  to X5 won't be a problem since the net name for GND still exists on the schematic. Any wires or vias given the GND name won't suddenly vanish if the GND to X5 is deleted. 
You can just copy an existing GND symbol to replace it.
From your previous post

> If I replace the GND connection from X5 in the Schematics, It looks like I get all GND connections on the board, just unrouted.

Were the ground planes visible (i.e. not black background)? If not, the GND connections appear as unrouted yellow lines. Check the layers are visible and use the command
```set polygon on; ratsnest```
The ground planes should appear and the unrouted gnd will become routed.
Are you new to Eaglecad?

**Comment by ghost** [comment]
> Are you new to Eaglecad?

Yes, I had the chance to get an education license, so I thought "Why not"?

> Were the ground planes visible (i.e. not black background)? If not, the GND connections appear as unrouted yellow lines. Check the layers are visible and use the command
set polygon on; ratsnest

Still not working. I toke some research and found the Olimex OLinuXino Boards. They have some with an i.MX233 (same processor architecture). The boards have only 64 MB of RAM but, for me, it's enough and they also use EAGLE.

**Comment by kinweilee** [comment]
I've built a board with imx233 too, the LQFP chip and SDRAM can be hand soldered. The olimex boards are a good place to start, but I advise looking at the NXP reference designs too. Good luck.
